modules/webgl/src/features/features.js

```diff
@@ -37,14 +36,36 @@ export function getFeatures(gl) {
 
 // TODO - cache the value
 function isFeatureSupported(gl, cap) {
+  gl.luma = gl.luma || {};
+  gl.luma.caps = gl.luma.caps || {};
+
+  let isSupported;
+
+  if (gl.luma.caps[cap] !== undefined) {
+    isSupported = gl.luma.caps[cap];
+  } else {
+    isSupported = queryFeature(gl, cap);
+    gl.luma.caps[cap] = isSupported;
+  }
+
+  if (!isSupported) {
+    log.log(LOG_UNSUPPORTED_FEATURE, `Feature: ${cap} not supported`)();
+  }
+
+  return isSupported;
+}
+
+function queryFeature(gl, cap) {
   const feature = WEBGL_FEATURES[cap];
   assert(feature, cap);
+  // Shouldn't be calling if it is already queried
+  assert(gl.luma.caps[cap] === undefined);
```

Comment:
This is unnecessary since this function is only called in one place (also an exception is overkill for something like this).

Reply:
removed.